### DISCUSSION TITLE
Ultralytics Live Sessions, Ep.1 - You Only Lose Your Notebook Once

### DISCUSSION THREAD

**Discussion by taliabender** [ultralytics/yolov5#10283] [Show and tell]
We are excited to announce [Ultralytics Live Sessions](https://www.youtube.com/@Ultralytics/streams)! 🚀

We'll stream our very first episode live on YouTube at https://youtu.be/KS4weDInJYs on **Tuesday, November 29th** with Victor Sonck of [ClearML](https://cutt.ly/yolov5-readme-clearml). Glenn and Victor will discuss all things experiment tracking and how you can maximize the benefits of our Ultralytics YOLOv5 x ClearML integration. 🔥

<a align="center" href="https://youtu.be/KS4weDInJYs" target="_blank"><img width="850" src="https://user-images.githubusercontent.com/85292283/203834465-fa67ead1-d87b-4212-8a1d-acfd4d8e7226.png"></a>

**Comment by glenn-jocher** [comment]
@taliabender awesome! I'm looking forward to this, and @thepycoder and I will be there chatting about YOLOv5 and the latest ClearML logging integration :)

**Reply by thepycoder** [reply]
Looking forward to it 😁
